File: src/main/java/tiagoanacleto/bookstore/model/Book.java
package tiagoanacleto.bookstore.model;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;

@Getter @Setter
@EqualsAndHashCode(onlyExplicitlyIncluded = true)
public class Book {

    @EqualsAndHashCode.Include
    private Long isbn;

    private String title;
    private String description;
    private String author;
    private String genre;
    private String pages;
    private String ageRange;
    private String price;
    private Long qty;

    @Override
    public String toString() {
        return title + "," + description + "," + isbn + "," + author + "," + genre + "," +
                pages + "," + ageRange + "," + price + "," + qty;
    }
}

File: src/main/kotlin/bookstore/model/Book.kt
package bookstore.model

data class Book(
    var isbn: Long = 0,
    var title: String = "",
    var description: String = "",
    var author: String = "",
    var genre: String = "",
    var pages: String = "",
    var ageRange: String = "",
    var price: String = "",
    var qty: Long = 0
)
